<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>发送信息</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务器地址</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QSpinBox" name="spinBox">
      <property name="minimum">
       <number>1000</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>端口</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actconn"/>
   <addaction name="actdis"/>
   <addaction name="actclear"/>
   <addaction name="actexit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actconn">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/conn.JPG</normaloff>:/images/conn.JPG</iconset>
   </property>
   <property name="text">
    <string>链接服务器</string>
   </property>
   <property name="toolTip">
    <string>链接服务器</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actdis">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/disconn.JPG</normaloff>:/images/disconn.JPG</iconset>
   </property>
   <property name="text">
    <string>断开链接</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actclear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/clear.bmp</normaloff>:/images/clear.bmp</iconset>
   </property>
   <property name="text">
    <string>清空文本框</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actexit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/exit.bmp</normaloff>:/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>推出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
